docs/source/tutorial_notebooks/getting_started_1_register_datasets.ipynb

```diff
@@ -73,11 +70,26 @@
     "datareg = DataRegistry()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "aa074027-5c6c-48d1-b79f-a6724a1bb386",
+   "metadata": {},
+   "source": [
+    "With no arguments, the `DataRegistry` class will automatically attempt to:\n",
+    "- establish a connection to the registry database using the information in your `~/.config_reg_access` and `~/.pgpass` files\n",
+    "- connect to the default database schema\n",
+    "- use the default NERSC \"`site`\" for the `root_dir`\n",
+    "\n",
+    "The root directory (`root_dir`) is the base path where all ingested data will be copied to. Other than for testing, this should generally be the NERSC `site` address."
```

Comment:
Maybe a bit clearer to say
"...is the base path under which all ingested data will be copied"
It's the "under which" that matters, to make it clear that we're not overwriting base path.